variants/heltec_wireless_paper/platformio.ini

```diff
@@ -5,4 +5,7 @@ build_flags =
   ${esp32s3_base.build_flags} -D HELTEC_WIRELESS_PAPER -I variants/heltec_wireless_paper
 lib_deps =
   ${esp32s3_base.lib_deps}
-  zinggjm/GxEPD2@^1.5.2
+  https://github.com/ixt/GxEPD2#39f325b677713eb04dfcc83b8e402e77523fb8bf
```

Comment:
This should probably be changed to latest commit on meshtastic's fork now, although it's no issue for it to track mine.